hack/trusted-tasks-update.sh

```diff
@@ -27,7 +27,7 @@ set -o pipefail
 mapfile -td ' ' COLLECT < <(echo -n "${COLLECT:-git oci}")
 INPUT_IMAGE=${INPUT_IMAGE:-quay.io/redhat-appstudio-tekton-catalog/data-acceptable-bundles:latest}
 OUTPUT_IMAGE=${OUTPUT_IMAGE:-$INPUT_IMAGE}
-GIT_REPOSITORY=git+https://github.com/redhat-appstudio/build-definitions.git
+GIT_REPOSITORY=git+https://github.com/konflux-ci/build-definitions.git
```

Comment:
@zregvart how much will moving this repo to github.com/konflux-ci break the trusted task data?

I suppose we'll need to allow both github.com/konflux-ci and github.com/redhat-appstudio for all the tasks? (If anyone is using the tasks with the git resolver today, they're using github.com/redhat-appstudio and they can keep using it thanks to the automatic redirects)

Reply:
Updated the script to add both urls as trusted, LMK what you think